openapi.json

```diff
@@ -272,31 +266,20 @@
           "status": {
             "type": "string"
           },
-          "process": {
-            "type": "string"
-          },
           "sites": {
-            "type": "array",
-            "items": {
-              "type": "object",
-              "properties": {
-                "server-name": {
-                  "type": "string"
-                },
-                "port": {
-                  "type": "string"
-                }
-              },
-              "required": [
-                "server-name",
-                "port"
-              ]
-            }
+            "type": "object",
+            "properties": {
+              "site-name": {
```

Comment:
where this property is used? can you show an example for JSON sending to the backend?

Reply:
it's actually a bit wrong
i've slightly changed the format of nginx data, and forgot to update the spec.
so "sites" is an array with the names of sites located on the server

the request itself looks like this:

{
        "hostname": "localhost.localdomain",
        "name": "nginx-info",
        "project_token": "321084yr1873y4813yr8ewyg3",
        "nginx-info":
        {
                "status": "active (running) since Thu 2019-12-05 17:21:46 MSK; 12min ago",
                "sites":
                [
                         "domain-one.com",
                         "example.com",
                         "test.com"
                ]
        }
}